### DISCUSSION TITLE
服务化部署使用高性能推理插件启动服务时报错

### DISCUSSION THREAD

**Discussion by 000px** [PaddlePaddle/PaddleX#2582] [Q&A]
<img width="809" alt="屏幕截图 2024-12-01 222936" src="https://github.com/user-attachments/assets/cecaba47-2c7a-4550-ab41-ba240eec2164">

不使用高性能推理插件可以正常启动，小白实在看不懂这个错误。😭
paddlex 3.0.0b2
python 3.10.9
Huawei Cloud EulerOS release 2.0 (West Lake)

**Comment by Bobholamovic** [comment]
建议尝试最新版本的paddlex～
